src/styles/base/_override.typography.scss

```diff
@@ -121,7 +130,7 @@
   font-size: $font-size-md;
   line-height: 24px;
   @media screen and (max-width: $width-md) {
-    font-size: $mct-p2-mobile-font-size !important;
+    font-size: $mct-p2-mobile-font-size;
```

Comment:
Since we are trying to move the mct specific styles out of the stylesheet can we [create a new variable in the variables](https://github.com/CMSgov/mgov-design-system/blob/cbf2c7c1f44d169a2c9d2814840c9a18dfb02595/src/styles/settings/_variables.type.scss) file for this font size?

Reply:
This can probably be taken out. I'll confirm with the Tiger Team but I don't think we are going to be changes font sizes for mobile.